ant/make-host.sh

```diff
@@ -1,234 +0,0 @@
-#!/bin/bash
-
-#================================================================#
-# Copyright (c) 2010-2011 Zipline Games, Inc.
-# All Rights Reserved.
-# http://getmoai.com
-#================================================================#
-
-	set -e
-
-	usage="usage: $0 -p <package> [-s] [-i thumb | arm] [-a all | armeabi | armeabi-v7a] [-l appPlatform] [--use-fmod true | false] [--use-untz true | false] [--disable-adcolony] [--disable-billing] [--disable-chartboost] [--disable-crittercism] [--disable-facebook] [--disable-push] [--disable-tapjoy]"
```

Comment:
We have never used this file and I always got confused with the other one so I just removed it